config/initializers/geocoder.rb

```diff
@@ -0,0 +1,30 @@
+Geocoder.configure(
+  # per Geocoder documentation, specifying a header while using default service Nominatim
+  # https://github.com/alexreisner/geocoder/blob/master/README_API_GUIDE.md#nominatim-nominatim
+  http_headers: { "User-Agent" => "steven@looseendsproject.org" }
```

Comment:
We'll still need to change this - I'd do info@looseendsproject.org if we need an email in there

Reply:
I would say that minimally. The boilerplate they offer is `Geocoder.configure(http_headers: { "User-Agent" => "your contact info" })`. I'd think email would be appropriate? Otherwise maybe just the website or "looseendsproject"